Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -460,13 +460,19 @@ extension SourceFilesList {
 }
 
 extension ResourceFileElements {
-    fileprivate static func from(resources: [PackageInfo.Target.Resource], path: AbsolutePath) -> Self? {
+    fileprivate static func from(resources: [PackageInfo.Target.Resource], path: AbsolutePath, excluding: [String]) throws -> Self? {
         let resourcesPaths = resources.map { path.appending(RelativePath($0.path)) }
         guard !resourcesPaths.isEmpty else { return nil }
-        return .init(resources: resourcesPaths.map { absolutePath in
-            let absolutePathGlob = absolutePath.extension != nil ? absolutePath : absolutePath.appending(component: "**")
-            return .glob(pattern: Path(absolutePathGlob.pathString))
-        })
+        return try .init(
```

Comment:
I think this is no longer throwing, isn't it? if it isn't, also the ResourceFileElements.from doesn't need to be throwing